Samples/UniversalDecoder/tools/copy-codecs.js

```diff
@@ -10,7 +10,8 @@ var args = process.argv.slice(2);
 const srcDir = args[0] || './node_modules/lorawan-devices/vendor';
 const dstDir = args[1] || './codecs';
 const indexFilePath = path.join(dstDir, 'index.js');
-
+// cube.js is ignored as esprima doesn't support class parsing https://github.com/jquery/esprima/issues/1971
+const ignoreList = ["greenme/cube.js"];
```

Comment:
I believe that using the `!(pattern|pattern|pattern)` with [`glob`](https://github.com/isaacs/node-glob) on the following line will give you the same effect.

Reply:
Amended thank you :)